cache/rediscache/rediscache_test.go

```diff
@@ -0,0 +1,204 @@
+package rediscache_test
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/terranodo/tegola/cache"
+	"github.com/terranodo/tegola/cache/rediscache"
+)
+
+// TestNew will run tests against a local redis instance
+// on 127.0.0.1:6379
+func TestNew(t *testing.T) {
+	if os.Getenv("RUN_REDIS_TESTS") != "yes" {
+		fmt.Println("RUN_REDIS_TESTS not set to 'yes' skipping tests")
+		return
+	}
+
+	type tc struct {
+		config map[string]interface{}
+		err    error
+	}
+
+	testcases := map[string]tc{
+		"redis explicit config": {
+			config: map[string]interface{}{
+				"network":  "tcp",
+				"address":  "127.0.0.1:6379",
+				"password": "",
+				"db":       0,
+				"max_zoom": 0,
+			},
+			err: nil,
+		},
+		"redis implicit config": {
+			config: map[string]interface{}{},
+			err:    nil,
+		},
+		"redis bad config":{
+			config: map[string]interface{}{
+				"address": "127.0.0.1:6000",
+			},
+			err: fmt.Errorf("dial tcp 127.0.0.1:6000: connect: connection refused"),
+		},
+	}
+
+	for i, tc := range testcases {
+		_, err := rediscache.New(tc.config)
+		if err != nil {
+			if tc.err != nil && err.Error() == tc.err.Error() {
```

Comment:
this would change to:
` if tc.errMatch != "" && strings.Contains(err.Error(),tc.errMatch) {`

doc for [strings.Contains](https://godoc.org/strings#Contains)

Reply:
Ah okay, that should fix it. Thank you.